pkg/parquetquery/iters.go

```diff
@@ -631,7 +631,7 @@ func (c *SyncIterator) seekRowGroup(seekTo RowNumber, definitionLevel int) (done
 			return true
 		}
 
-		if CompareRowNumbers(definitionLevel, seekTo, max) != -1 {
+		if CompareRowNumbers(definitionLevel, seekTo, *max) != -1 {
```

Comment:
Do we want to check for nilness before dereferencing min/max?

Reply:
good call. added nil checks.